### DISCUSSION TITLE
开发者好，OCR出问题了

### DISCUSSION THREAD

**Discussion by andydznj** [shuax/MouseInc.Settings#346] [General]
有几天不能使用了，谢谢。

**Comment by andydznj** [comment]
目前已经升级到 2.13. 0 A

**Reply by yinlsea** [reply]
2.13在什么地方下载？

**Reply by uiop321** [reply]
目前正常使用，你看看往前的内容解答。

**Reply by sunniestboy** [reply]
2.13. 0 A 有下载地址吗， 2.13. 0依然还是OCR用不了。

**Reply by andydznj** [reply]
> 2.13在什么地方下载？

自动升级的

**Reply by andydznj** [reply]
> 目前正常使用，你看看往前的内容解答。

不行啊

**Comment by Eunknight** [comment]
https://github.com/shuax/MouseInc.Settings/discussions/335#discussioncomment-3523123
